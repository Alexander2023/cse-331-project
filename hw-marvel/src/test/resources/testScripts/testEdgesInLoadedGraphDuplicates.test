# Tests that duplicates in csv file are not added to the loaded graph

LoadGraph smallWithDuplicates smallWithDuplicates.csv

ListChildren smallWithDuplicates c1
ListChildren smallWithDuplicates c2