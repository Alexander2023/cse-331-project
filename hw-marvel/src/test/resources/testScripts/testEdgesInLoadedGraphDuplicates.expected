# Tests that duplicates in csv file are not added to the loaded graph

loaded graph smallWithDuplicates

the children of c1 in smallWithDuplicates are: c2(b1)
the children of c2 in smallWithDuplicates are: c1(b1)